index.html

```diff
@@ -46,7 +46,7 @@
       }
       main > * { grid-column: content; }
```

Comment:
`grid-column: content` is not standard css, is it a feature of some library?

If I understand the goal correctly, we want every child of main to be placed in the grid area labelled `content`? Perhaps this should be written as `grid-column: content-start / content-end;` then.

Reply:
> grid-column: content is not standard css, is it a feature of some library?

It's is just a shorthand for `grid-column: content-start / content-end`.

It makes more sense with more content (screenshot from original draft):

![2025-02-21 10-21-21 Arc - Act different@2x](https://github.com/user-attachments/assets/21cec035-4c6a-46e5-9850-bd203ea040f0)

Intent is to limit everything to a center column _and_ show the title as wide as the viewport permits.